platforms/karaf/features/src/main/resources/features.xml

```diff
@@ -1244,6 +1244,17 @@
     <bundle>mvn:org.apache.camel/camel-jgroups/${project.version}</bundle>
     <bundle dependency='true'>mvn:org.jgroups/jgroups/${jgroups-version}</bundle>
   </feature>
+  <feature name='camel-jgroups-raft' version='${project.version}' resolver='(obr)' start-level='50'>
+    <feature version='${project.version}'>camel-core</feature>
+    <bundle dependency='true'>mvn:commons-io/commons-io/${commons-io-version}</bundle>
+    <bundle>mvn:org.apache.camel/camel-jgroups-raft/${project.version}</bundle>
```

Comment:
In this case I guess dependency='true' is needed.

Reply:
Which one? commons-io?

Reply:
The camel-jgroups one

Reply:
done!